lfucg-exactions/server/notes/viewsets.py

```diff
@@ -22,6 +22,8 @@ def get_queryset(self):
         child_object_id = self.request.query_params.get('object_id', None)
         parent_content_type_string = self.request.query_params.get('parent_content_type', None)
         parent_object_id = self.request.query_params.get('parent_object_id', None)
+        grandparent_content_type_string = self.request.query_params.get('grandparent_content_type', None)
```

Comment:
Better to handle figuring out if there are notes on the plat or subdivision entirely on the backend. That way client code is simpler.